frontend/src/pages/home/aiFlows/CreateAndTrainGallery.tsx

```diff
@@ -11,6 +11,17 @@ const CreateAndTrainGallery: React.FC<{ onClose: () => void }> = ({ onClose }) =
 
   const infoItems = [];
 
+  const workbenchDescriptionTrailer = pipelinesAvailable ? (
```

Comment:
Could do this inline or at least put it inside the if-statement. `consts` are not like `vars` -- its reasonable to lexical scope them to their use-cases.